config.lua

```diff
@@ -203,3 +208,45 @@ function BerserkAddon:ToggleShowInChat_1(info, value)
         self:CancelThicker()
     end
 end
+
+
+------------------------------
+-- Utility Functions
+------------------------------
+function BerserkAddon:printHelp()
+    print("Berserk CLI Options\n",
```

Comment:
So instead of using regular `print()` we could use `self:Print()` method that belongs to `AC3` library. The difference is that using ac3 the message will contain`Berserk Addon: ` at the beginning.

`self:Print("hello world!")`  will pront the message below
`BerserkAddon: hello world`

Reply:
The only problem with this is that it formats stuff weird. With regular lua print it looks normal but with AC3 print it looks like:

Mage Rotation:
  Mages:
On bench:
  Mage:

Basically it puts a colon after every line. Is there a way to make it not do that?

Reply:
not sure if I have the same issue locally
```
    self:Print("This week's mage rotation:\n",
        r);
    self:Print("On bench:\n",
        b);
```

![image](https://user-images.githubusercontent.com/1949867/99919378-094e5f80-2ceb-11eb-9dab-9992c0a2700c.png)

Reply:
Ah before I did ```self.Print()``` and not ```self:Print()```
However the formatting is still not ideal. I'll play with it some more